prediction-applier/src/main/resources/application.yml

```diff
@@ -11,7 +11,12 @@ camel:
 hyland-experience:
   insight:
     prediction:
-      endpoint: aws2-sqs://arn:aws:sqs:us-east-1:000000000000:hxinsight-prediction-queue
+      endpoint: direct:test
+    prediction-listener:
+      poll-period: 300_000 #5 min
+      predictions-source-endpoint: direct:mocked
```

Comment:
Here also it doesn't seem right

Reply:
we don't really have any service we could connect to so I would leave it as it is (and maybe setup some wiremock when it'll be needed)